### DISCUSSION TITLE
Controlnet

### DISCUSSION THREAD

**Discussion by IIWstudio** [danilodelucio/NukeDiffusion#2] [Ideas]
Thanks a ton for your amazing work! It's working like a charm. Quick question for you: how feasible do you think it would be to integrate controlnet modules into stable diffusion to push the concept even further?
Having controlnet integration could really unlock the full potential of using this kind of tool for DMP work. 

Beside I'm actually thinking of tweaking the gizmo on my end to have it write the input image and mask by itself, and set up the most recent generated image as the read output. I believe this would be a more than welcome addition to streamline the workflow.

Thank youuu

**Comment by danilodelucio** [comment]
Hi there, thank you so much for your feedback, I'm glad that is working and you are enjoying the tool! 😍

I'm interested in adding ControlNet and other stuff, but honestly, I don't know how long I will keep working on it. Since I am developing it by myself and in my spare time, I don't know how much time would take to develop all these features because it's more advanced controls of postprocessing. I need to study the documentation again, then I see if I can implement it for what I've done. 

Regarding the exporting images, I have something already done in my code but I couldn't make it for this version, I hope for the next release! 😅

Thanks for leaving a comment and being interested, feel free to reach out any time!
